pkg/orchestration/wiring/kubeingress/plugin_test.go

```diff
@@ -56,31 +136,39 @@ func TestExtractKubeComputeDependency(t *testing.T) {
 	t.Run("valid single dependency", func(t *testing.T) {
 		deps := []wiringplugin.WiredDependency{computeDep}
 
-		res, _, err := extractKubeComputeDependency(deps)
+		res, _, err := extractKubeComputeDependency(&deps)
 		assert.NoError(t, err)
-		assert.ObjectsAreEqual(deploymentObj, res)
+		assert.Equal(t, deploymentObj, res)
```

Comment:
Previous assertion did nothing because ObjectsAreEqual returned a boolean and didn't actually do an assertion